src/main/kotlin/pl/newbies/event/application/model/EventRequest.kt

```diff
@@ -40,8 +51,14 @@ data class AddressRequest(
 
     init {
         validate(this) {
-            validate(AddressRequest::city).isNotBlank()
-            validate(AddressRequest::place).isNotBlank()
+            validate(AddressRequest::city)
+                .isNotBlank()
+                .hasSize(max = 50)
+                .oneLine()
+            validate(AddressRequest::place)
+                .isNotBlank()
+                .hasSize(max = 50)
+                .oneLine()
```

Comment:
I'm not sure if one line in place is enough to store address 
we don't have full address  combo in database
only city and place

Reply:
But do we need more? We can omit postal code and everything else cause user can pick coordinates on map, I think that this field will be used more like "how to get there" tip. What do you suggest?

Reply:
i think "how to get there" field would greatly profit from multi line "how to get there" instructions

Reply:
Done